doc/devel/release_guide.rst

```diff
@@ -455,22 +455,20 @@ Due to branch protections for the ``main`` branch, this is merged via a standard
 request, though the PR cleanliness status check is expected to fail. The PR should not
 be squashed because the intent is to merge the branch histories.
 
-Announcing
-==========
-
-The final step is to announce the release to the world.  A short
-version of the release notes along with acknowledgments should be sent to
-
-- matplotlib-users@python.org
-- matplotlib-devel@python.org
-- matplotlib-announce@python.org
+Publicize release
+=================
 
-In addition, announcements should be made on social networks (e.g., Twitter via the
-``@matplotlib`` account, any other via personal accounts).
+The final step is to announce the release to the world:
```

Comment:
When a new Matplotlib release is published, it should be announced through our communcation channels: